examples/Tweet.hs

```diff
@@ -108,7 +97,10 @@ main = runBH' $ do
   let boost = Nothing
```

Comment:
The `putTemplate` is failing as well at the moment with `Root mapping definition has unsupported parameters:  [location : {type=geo_point}]\"}]`

I could not figure out how to fix it, my only working fix was to just leave the list of `TweetMapping` empty..